service/src/main/java/bio/terra/axonserver/app/controller/GetFileController.java

```diff
@@ -66,8 +71,23 @@ public ResponseEntity<Resource> getFileInBucket(
 
     BearerToken token = getToken();
 
+    HttpRange byteRange = getByteRange();
+
     Resource resourceObj =
-        fileService.getFile(token, workspaceId, resourceId, objectPath, convertTo);
+        fileService.getFile(token, workspaceId, resourceId, objectPath, convertTo, byteRange);
     return new ResponseEntity<>(resourceObj, HttpStatus.OK);
   }
+
+  private HttpRange getByteRange() {
```

Comment:
i don't understand why you can't put this in the query parameter?

Reply:
Can you elaborate? This method makes an HttpRange by parsing the serverletRequest's range header value (which is a string).

Reply:
just curious how you made the decision to use a header instead of queryparam. 

i think we usually use query param in wsm but I guess this is an internet standard to use header since range is a standard http header?